adaptors/library/jobs/satusehat-create-condition.js

```diff
@@ -0,0 +1,97 @@
+// Create a diagnosis in satusehat
+
+fn(state => {
+  state.conditionMappedData = state.visit.diagnosis.flatMap(diagnosis =>
```

Comment:
I would probably call this `state.conditions`. I can guess what a conditions array is - but I'm not sure what a `conditionMappedData` thing is